example/glue/ScavengerRootScanner.hpp

```diff
@@ -56,6 +58,7 @@ class MM_ScavengerRootScanner : public MM_Base
 	void
 	scavengeRememberedSet(MM_EnvironmentStandard *env)

```

Comment:
This flush was carried forward from preOMR implementation (ie, for Java GC). I agree that all threads (master & helper) _should_ have empty RS buffers at this point. But GC_OMRVMInterface::flushCachesForGC() is called on master thread very early (in preCollect()) and it is not easy to verify from code inspection that there are no exceptional cases. I thought it would be safer to carry it forward, since if RS buffers are empty this flush has negligible performance impact.